extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -191,6 +195,25 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
     }
   };
 
+  public changeReplyOption = async (option: ReplyOptions) => {
+    if (!this.view.replyParams) {
+      return;
+    }
+    this.view.recipientsModule.clearRecipients();
+    if (option === 'a_forward') {
+      await this.view.quoteModule.addTripleDotQuoteExpandFooterAndQuoteBtn(this.view.replyMsgId, 'forward');
+    } else {
+      const recipients: Recipients = { to: this.view.replyParams.to, cc: this.view.replyParams.cc, bcc: this.view.replyParams.bcc };
+      this.view.recipientsModule.addRecipients(recipients, false).catch(Catch.reportErr);
```

Comment:
We should use `this.view.replyParams` directly, as the new object still has references to the old object's arrays, and we would need to care about all the places they could be possibly modified.
`this.view.recipientsModule.addRecipients(this.view.replyParams, ...`